### ISSUE TITLE
Bug: download fails for videos with no trickplay (seek thumbnails) - Failed to match uuid

### ISSUE THREAD

**Issue by NTFSvolume** [MiyukiQAQ/MissAV-Downloader#19]
Example url:

https://missav.com/dm57/en/EBOD-440

Command: 

```
miyuki -urls "https://missav.com/dm57/en/EBOD-440" -ffmpeg
```

Output:
```

Miyuki - 2024-09-14 08:23:14 - INFO - Processing URL: https://missav.com/dm57/en/EBOD-440
Miyuki - 2024-09-14 08:23:14 - ERROR - Failed to match uuid.
Miyuki - 2024-09-14 08:23:14 - INFO - Processing URL Complete: https://missav.com/dm57/en/EBOD-440

```

This video does not have trickplay available (seek previews from the `sixyik` server), so the `get_movie_uuid` function fails.

checking the source, the uuid could be parsed from the javascript code of the source variable:

![javascript source code](https://github.com/user-attachments/assets/0111f763-c569-4a60-b23f-cf2f181fc2b9)

![console_output](https://github.com/user-attachments/assets/79cd4d3b-9860-4f0f-a802-cfee13a63742)

basically, each section of the uuid is written using "|" as separator, after the sources variables names (in reverse).

**Comment by MiyukiQAQ** [comment]
> Example url:
> 
> https://missav.com/dm57/en/EBOD-440
> 
> Command:
> 
> ```
> miyuki -urls "https://missav.com/dm57/en/EBOD-440" -ffmpeg
> ```
> 
> Output:
> 
> ```
> 
> Miyuki - 2024-09-14 08:23:14 - INFO - Processing URL: https://missav.com/dm57/en/EBOD-440
> Miyuki - 2024-09-14 08:23:14 - ERROR - Failed to match uuid.
> Miyuki - 2024-09-14 08:23:14 - INFO - Processing URL Complete: https://missav.com/dm57/en/EBOD-440
> ```
> 
> This video does not have trickplay available (seek previews from the `sixyik` server), so the `get_movie_uuid` function fails.
> 
> checking the source, the uuid could be parsed from the javascript code of the source variable:
> 
> ![javascript source code](https://private-user-images.githubusercontent.com/172021377/367520254-0111f763-c569-4a60-b23f-cf2f181fc2b9.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjYzMjY1NDUsIm5iZiI6MTcyNjMyNjI0NSwicGF0aCI6Ii8xNzIwMjEzNzcvMzY3NTIwMjU0LTAxMTFmNzYzLWM1NjktNGE2MC1iMjNmLWNmMmYxODFmYzJiOS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwOTE0JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDkxNFQxNTA0MDVaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0zNjNhZmFiZDAzMTQ1YWY2MGMzODQ5MzEyMzIwYjA4MTdmNjhkMGMwMmM4OGJlMDQ3MTllMmJhMTIzMDJhOTE2JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.rUjj1xdaEQG9AJAXjvetQRYamIKJdyCVDwn00yZ7BiQ)
> 
> ![console_output](https://private-user-images.githubusercontent.com/172021377/367519813-79cd4d3b-9860-4f0f-a802-cfee13a63742.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjYzMjY1NDUsIm5iZiI6MTcyNjMyNjI0NSwicGF0aCI6Ii8xNzIwMjEzNzcvMzY3NTE5ODEzLTc5Y2Q0ZDNiLTk4NjAtNGYwZi1hODAyLWNmZWUxM2E2Mzc0Mi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwOTE0JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDkxNFQxNTA0MDVaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT05ZjEwNTJkMzRiMmFlZjIzZjdmYTZiMzY2MGExNzcxMTc0MTE2M2Q3OTUwMDcwZDRmNGQ3NGNjNjk1YmZkNWZiJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.46v2GUpe1Lc_ac0S1EpPno9YriRfG8Vjg6LJ4SJF1s0)
> 
> basically, each section of the uuid is written using "|" as separator, after the sources variables names (in reverse).

Thank you very much, the information you provided was very helpful. This bug has now been fixed.
